docs/references/backend/overview.mdx

```diff
@@ -225,6 +225,16 @@ Phone number operations are exposed under `clerkClient.phoneNumbers`. The follow
 
 - [`deletePhoneNumber()`](/docs/references/backend/phone-numbers/delete-phone-number)
 
+## Testing tokens operations
+
+A testing token can be used to bypass the Bot Protection rules on E2E testing.
+
+Testing tokens operations are exposed under `clerkClient.testingTokens`. The following methods are available:
+
+### Testing tokens `create` operations
+
+- [`createTestingToken()`](/docs/references/backend/testing-tokens/create-testing-token)
```

Comment:
Will we be adding more items to this list in this PR?

Reply:
@S3Prototype no we are not going to add anything else